src/components/Review.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div :class="$style['wrapper']">
```

Comment:
It is a 'div hell'! Check it if you can remove some excess divs or not. I think you can do this

Reply:
yeah ... there is a div hell but I think it could be better not to effort on cleaning this code because it is possible for us to remove it from source code 
we must wait for feedback